Add gosec to kubeops

antgamdia opened the pull request:
### Description of the change

After reporting #4810 , I've added the linter just in the `kubeops` code . 
The building is gonna fail now, but if we start fixing the issues, we will be, eventually, able to merge this PR.

### Benefits

Prevent new `gosec` issues.

### Possible drawbacks

N/A

### Applicable issues

- related #4810 

### Additional information

N/A

castelblanque commented:
> The building is gonna fail now

It seems that CI passed fine. Is it OK like this or should have it failed?

antgamdia commented:
> It seems that CI passed fine. Is it OK like this or should have failed?

Sorry, I copy&pasted the same msg but then I didn't edit it when I added the fixes (example https://github.com/vmware-tanzu/kubeapps/pull/4992/commits/2c51f5110c0146dade3e1231dd548b9331ef29b5). 
In short, it should have failed when I created the PR (and so it happened), but once I added the fixes, it should have passed.
